pkg/agent/client/client.go

```diff
@@ -43,7 +43,12 @@ var (
 	}
 )
 
-const rpcTimeout = 30 * time.Second
+const (
+	rpcTimeout = 30 * time.Second
+	// We can be more aggressive with timeouts in cases where a valid SVID
+	// exists in the cache but is old enough to try for a new SVID quickly
+	rpcTimeoutWithCacheHit = 1 * time.Second
```

Comment:
We'll want this to be a bit higher, 5 or at most 3 seconds. The higher we make the timeout the more likely it will be that someone will be hit by thy issue, but we also don't want it to be too low to make it impossible to get a SVID.

Reply:
I've increased to 5 seconds